### DISCUSSION TITLE
RKE2 generating false positive result

### DISCUSSION THREAD

**Discussion by nkhader7** [aquasecurity/kube-bench#1584] [Questions and Help]
I used the following command ./kube-bench --config-dir /XXX/kube-bench/cfg/  --config /XXXX/kube-bench/cfg/config.yaml  --benchmark rke2-cis-1.7 for RKE2 v1.26.4 and getting the false positive results 1.1.1 run the below command chmod600 but the file permissions are 600. please help

**Comment by andypitcher** [comment]
@nkhader7 could you run the same command again with the this option `-v7` and provide the output ? For reference, here is the link to [1.1.1](https://github.com/aquasecurity/kube-bench/blob/0f8dfaf1156652b556c913aa8e78fccbb1dfd186/cfg/rke2-cis-1.7/master.yaml#L11).
